generator/src/test/java/com/scottlogic/deg/generator/cucumber/MatchingRegex.feature

```diff
@@ -259,145 +259,156 @@ Scenario: Running a 'matchingRegex' for a minimum length that is less zero shoul
+         | foo  |
+         | null |
 
-@ignore
-Scenario: Running a 'matchingRegex' request alongside an ofType = temporal should fail with an error message
-       Given foo is matching regex /[a]{1}/
+Scenario: Running a 'matchingRegex' request alongside an ofType = temporal should generate null
+     Given foo is matching regex /[a]{1}/
        And foo is of type "temporal"
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be generated:
+       | foo  |
+       | null |
 
 Scenario: Running a 'matchingRegex' request alongside a non-contradicting matchingRegex constraint should be successful
-       Given foo is matching regex /[a-z]{1,3}/
+     Given foo is matching regex /[a-z]{1,3}/
        And foo is matching regex /[b]{2}/
      Then the following data should be generated:
        | foo  |
        | null |
        | "bb" |
 
-Scenario: Running a 'matchingRegex' request alongside a contradicting matchingRegex constraint should fail with an error message
-       Given foo is matching regex /[a]{1}/
+  @ignore #may need to raise issue
+Scenario: Running a 'matchingRegex' request alongside a contradicting matchingRegex constraint should generate null
+     Given foo is matching regex /[a]{1}/
        And foo is matching regex /[b]{2}/
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be generated:
+       | foo  |
+       | null |
 
 Scenario: Running a 'matchingRegex' request alongside a non-contradicting containingRegex constraint should be successful
-       Given foo is matching regex /[b]{2}/
+     Given foo is matching regex /[b]{2}/
        And foo is containing regex /[a-z]{1,3}/
      Then the following data should be generated:
        | foo  |
        | null |
        | "bb" |
 
-Scenario: Running a 'matchingRegex' request alongside a contradicting containingRegex constraint should fail with an error message
-       Given foo is matching regex /[b]{3}/
+  @ignore #may need to raise issue
+Scenario: Running a 'matchingRegex' request alongside a contradicting containingRegex constraint should generate null
+     Given foo is matching regex /[b]{3}/
        And foo is containing regex /[a]{1,2}/
-     Then I am presented with an error message
-       And no data is created
+     Then the following data should be generated:
+       | foo  |
+       | null |
 
 Scenario: Running a 'matchingRegex' request alongside a non-contradiction ofLength constraint should be successful
-       Given foo is matching regex /[b]{2}/
+     Given foo is matching regex /[b]{2}/
        And foo is of length 2
      Then the following data should be generated:
        | foo  |
        | null |
        | "bb" |
 
-Scenario: Running a 'matchingRegex' request alongside a contradicting ofLength (too short) constraint should fail with an error message
-       Given foo is matching regex /[b]{2}/
+  @ignore #may need to raise issue
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501